jenkins/databricks/create.py

```diff
@@ -33,20 +33,21 @@ def main():
   num_workers = 1
   worker_type = 'g4dn.xlarge'
   driver_type = 'g4dn.xlarge'
+  cluster_type = 'aws'
```

Comment:
Add 'cluster_type' parameter to make create.py common for aws and azure Databricks